### PR TITLE
Create tests

### LINKED ISSUE #10: [3pt]Add tests for the Food list
Add tests for the Food list

### LINKED ISSUE #11: [3pt]Add tests for General shopping list
Add tests for General shopping list

### PR DESCRIPTION
**Pull request by Ismailco**
- [x] Create end-to-end tests for The food page
- [x] Create end-to-end tests for The general shopping list page
#### tests on the system folder

### CODE REVIEW

File: spec/factories/users.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    sequence(:email) { |n| "hello+#{n}@test.com" }
+    password { 'password' }
+    password_confirmation { 'password' }
+  end
+end
```
Comment (ismailakinkunmi) [inline spec/factories/users.rb:7]: RSpec.describe 'recipes#index', type: :feature do
  describe 'recipe food' do
    before(:each) do

end

**Review by ismailakinkunmi — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @Ismailco , 

Good job so far!

- [ ] Your test file doesn't indicate Rspec and according to the project requirement RSpec and Capybara is required for testing kindly use Rspec
- [ ] kindly perform tests on food page not user

Cheers and Happy coding! 👏👏👏

File: spec/factories/users.rb (hunk 1 above)
Comment (Ismailco) [reply spec/factories/users.rb:7]: Hi @ismailakinkunmi,

I did include RSpec and used capybara for end-to-end testing on the Food page, The code you're showing me is for authentication so the test can access to food page if I don't include the FactoryBot to create a user. the test will be redirected to the sign_in page.
The file for the tests is in the system folder under the name `foods_spec.rb`, kindly recheck this,

Comment (Ismailco) [reply spec/factories/users.rb:7]: Screenshot_20220803_111855

Comment (ismailakinkunmi) [reply spec/factories/users.rb:7]: @Ismailco ok noted I re-run and get back to you

**Comment by Ismailco** [comment]
@ismailakinkunmi,
 I added all the tests to this PR, so there is only one PR for tests. good advice, thank you!

**Review by ismailakinkunmi — APPROVED** [review]
@Ismailco,

### Congratulation 🎉🎉🎉
Your project is complete nothing else than to merge it.

Cheers and Happy coding! 👏👏👏

**Pull request merged**